src/Parser/DeathKnight/Blood/CombatLogParser.js

```diff
@@ -20,6 +20,7 @@ import RunicPowerTracker from './Modules/RunicPower/RunicPowerTracker';
 
 import T20_2pc from './Modules/Items/T20_2pc';
 import T20_4pc from './Modules/Items/T20_4pc';
+import SkullflowersHaemostasis from './Modules/Items/Skullflowers_Haemostasis';
```

Comment:
Please move the underscore from the .js file name, just to match naming scheme.

Reply:
fixed